api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -677,17 +678,19 @@ def _update_venues_criteria(
     return changed_venues
 
 
-REMOVE_PRICING_POINT_TITLE = "Détacher les points de valorisation et remboursement"
-
-
 def _update_permanent_venues(venues: list[offerers_models.Venue], is_permanent: bool) -> list[offerers_models.Venue]:
     venues_to_update = [venue for venue in venues if venue.isPermanent != is_permanent]
     for venue in venues_to_update:
         venue.isPermanent = is_permanent
+        if is_permanent and venue.thumbCount == 0:
+            transactional_mails.send_permanent_venue_needs_picture(venue)
```

Comment:
Un **détail** sur du code déjà présent avant ta PR : ça me semblerait plus facilement compréhensible avec une seule itération de boucle : 
```suggestion
    for venue in venues:
        if venue.isPermanent != is_permanent:
            venue.isPermanent = is_permanent
            if is_permanent and venue.thumbCount == 0:
                transactional_mails.send_permanent_venue_needs_picture(venue)
```
Mais c'est une question de goût, l'optimisation de performance est microscopique :)

Reply:
Mais le filtrage dans la définition de venues_to_update me semble nécessaire ensuite, puisque on ne cherche qu'à retourner les lieux modifiés seulement, pour ne synchroniser que ceux-ci (cf l'usage de `_update_permanent_venues` dans la suite du code)

Reply:
Hmmm... tu as raison :)